PROJECT_PROGRESSION.md

```diff
@@ -203,13 +173,15 @@ Impact Stage projects are expected to participate actively in CPC proceedings, a
 
 #### Acceptance Criteria
 
-To graduate from At Large or Growth status, or for a new project to join as an Impact project, a project must meet the Growth stage criteria plus:
+To graduate from At Large stage or for a new project to join as an Impact project, a project must meet the following criteria:
 
+ * Document that it is being used successfully in production by at least two independent end users which, in the CPC’s judgement, are of adequate quality and scope.
  * Have a defined governing body of at least 5 or more members (owners and core maintainers), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer. 
- * Have a documented and publicly accessible description of the project's governance, decision-making, and release processes.
+ * Have a documented and publicly accessible description of the project's governance, decision-making, contribution, and release processes. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and AUTHORS.md file showing the current and emeritus committers.
+ * Demonstrate a substantial ongoing flow of commits and merged contributions.
  * Have a healthy number of committers from at least two organizations. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
  * Adopt the Foundation Code of Conduct.
- * Explicitly define a project governance and committer process. This is preferably laid out in a GOVERNANCE.md file and references a CONTRIBUTING.md and OWNERS.md file showing the current and emeritus committers.
+ * Accept security reports and publicly disclose them after the fix has been made available.
```

Comment:
I'm going to presume that we would not use a lack of security issues as a blocker for Impact, provided that the policies are in place for dealing with such. While it's of course likely that something like that will be encountered by a project before it reaches this level, I could well imagine such issues having all been caught at an early enough stage to not need to follow any special procedures for dealing with them.

I don't think this language needs to change, I just wanted to note here that this exact verbiage should not be understood to require a security report to have been made in the project.

Reply:
I read it as the project needs to have a process to accept security reports and handle them but no requirement that the process have been exercised.